auth/libraries/pip-auth/authlib/validator.py

```diff
@@ -99,7 +102,7 @@ def __verify_ticket_from_auth_service(
         """
         self.debug_fn("Validator.__verify_ticket_from_auth_service")
         response = self.http_handler.send_internal_request(
-            "/api/authenticate", headers=headers, cookies=cookies
+            "/api/authenticate", headers=headers, cookies=cookies, payload={}
```

Comment:
This should not be needed, if the parser accepts an empty payload (`null`, `undefined` `""`) and interpret it as `{}`.